docs/quickstart/install-and-use-a-package-in-visual-studio.md

```diff
@@ -1,78 +1,86 @@
 ---
-title: Install and use a NuGet package in Visual Studio
-description: A walkthrough tutorial on the process of installing and using a NuGet package in a Visual Studio project.
+title: "Quickstart: Install and use a NuGet package in Visual Studio (Windows only)"
+description: In this quickstart, you learn how to install and use a NuGet package in a Visual Studio project.
 author: JonDouglas
 ms.author: jodou
-ms.date: 07/24/2018
+ms.date: 08/19/2022
 ms.topic: quickstart
 ---
 
-# Quickstart: Install and use a package in Visual Studio (Windows only)
+# Quickstart: Install and use a NuGet package in Visual Studio (Windows only)
 
-NuGet packages contain reusable code that other developers make available to you for use in your projects. See [What is NuGet?](../What-is-NuGet.md) for background. Packages are installed into a Visual Studio project using the NuGet Package Manager, the [Package Manager Console](../consume-packages/install-use-packages-powershell.md), or the [dotnet CLI](install-and-use-a-package-using-the-dotnet-cli.md). This article demonstrates the process using the popular [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) package and a Windows Presentation Foundation (WPF) project. The same process applies to any other .NET or .NET Core project.
+*NuGet* packages contain Microsoft-supported reusable code that other developers have made available to you for use in your projects. You can install NuGet packages in a Microsoft Visual Studio project by using the NuGet Package Manager, the Package Manager Console, or the .NET CLI. This article demonstrates how to create a Windows Presentation Foundation (WPF) project with the popular `Newtonsoft.Json` package. The same process applies to any other .NET or .NET Core project.
```

Comment:
> *NuGet* packages contain Microsoft-supported reusable code that other developers have made available to you for use in your projects

I don't imagine we want to say they contain Microsoft-supported reusable code here do we?
It could be interpreted to imply that all packages are supported by MS which isn't true.

Reply:
Is replacing `dotnet` with `.NET CLI` in this case the correct thing? `dotnet` refers to the exact CLI command, while `.NET` is just a generic name for everything .NET.

Reply:
I agree "Microsoft-supported" should be removed. .NET CLI looks like the official name for the tool, so in this case, it looks good. I like having the link though.

Reply:
What about `nuget.exe`?

Reply:
nuget.exe's installation into project is not great, so probably a good idea not to recommend it tbh.

Reply:
We're discouraged from using links in quickstarts, especially in the intro paragraphs, so that the user stays focused. I could make an exception, though.
If we're not recommending nuget.exe/nuget.org, should I remove the TIP near the beginning? (To find a NuGet package, start with *nuget.org*... )

Reply:
We wouldn't want to recommend nuget.exe in our quickstart here. VS and dotnet.exe support full package installation.

Nuget.org is the service that provides packages so that's ok.